docs/dg/dev/upgrade-and-migrate/upgrade-to-php-81.md

```diff
@@ -8,6 +8,13 @@ redirect_from:
 
 ---
 
+{% info_block warningBox "Warning" %}
+
+As of September 2024, we no longer provide support for PHP 8.1 modules. To ensure continued compatibility, security, and access to new features,
+we highly recommend [upgrading to PHP 8.2](/docs/dg/dev/upgrade-and-migrate/upgrade-to-php-82.html). 
```

Comment:
Are you linking to a document that doesn't exist?

Reply:
This is a leftover, we got rid of that doc. Fixed, thanks.